src/pages/Dashboard.tsx

```diff
@@ -79,6 +77,12 @@ const useDashboardData = () => {
 
 export const Dashboard = () => {
 
+    /**
```

Comment:
This file contains some temporary code to insert desired KSM account and TODOs that can be ignored.

This file ALSO contains some UI changes that make the BSX crowdloan dashboard look like this:

![image](https://user-images.githubusercontent.com/25761350/141863987-6c0aff0e-a2f1-41e7-a276-fbce9954752a.png)

Thoughts?

Reply:
yeah it kinda makes sense to get rid of the minimal BSX reward, can you put like a diagonal stroke through the box under `change your account`, or fill it out somehow so its not just standing empty? or a dash in it, something.

Reply:
![image](https://user-images.githubusercontent.com/25761350/142025519-e38309e1-060c-4653-b8d7-71920d8b931d.png)

❓

Reply:
@maht0rz 👆